framework/include/base/Assembly.h

```diff
@@ -548,6 +550,7 @@ class Assembly
   void setResidualNeighbor(NumericVector<Number> & residual, TagID tag_id = 0);
 
   void addJacobian();
+  inline void addJacobianCoupledVarPair(MooseVariableBase * ivar, MooseVariableBase * jvar);
```

Comment:
Does the "inline" declaration really go here? I thought it goes later in the file where the function is actually defined.